spec/tags/whatsapp.md

```diff
@@ -34,3 +34,26 @@ When you receive a message from one contact, the sender and recipient is inverte
 * Sender: is the phone number of contact
 
 In API the sender is the field `from` and the receiver is the field `to` of message object.
+
+
+## Type of Messages
+
+* **Template Message**
+
+**What's the template message?**
+
+Message Templates are message formats for common reusable messages a business may want to send. Businesses must use Message Templates for sending notifications to customers.
+
+This allows a business to send just the template identifier along with the appropriate parameters instead of the full message content.
+
+References: <a href="https://developers.facebook.com/docs/whatsapp/message-templates/" target="_blank">WhatsApp Business API</a>
+
+**How can I purchase a new template of WhatsApp?**
+
+If you already have a WhatsApp business account just send email to *whatsapp@zenvia.com*
+
+* **Session Message**
```

Comment:
I don't think we should we use "Session Message", since its not how we call them on the API, and it is not also how they are called on the WhatsApp docs.